sdk/communication/Azure.Communication.CallingServer/src/Models/CallMediaRecognizeOptions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary>
+    /// Optional elements for Recognize.
+    /// </summary>
+    public class CallMediaRecognizeOptions
+    {
+        /// <summary>
+        /// Initializes a RecognizeOptions object.
+        /// </summary>
+        /// <param name="recognizeInputType"></param>
+        /// <param name="recognizeConfigurations"></param>
+        public CallMediaRecognizeOptions(RecognizeInputType recognizeInputType, RecognizeOptions recognizeConfigurations)
+        {
+            RecognizeInputType = recognizeInputType;
+            RecognizeOptions = recognizeConfigurations;
+        }
+        /// <summary>
+        /// Recognize Input Type.
+        /// </summary>
+        public RecognizeInputType RecognizeInputType { get; }
+
+        /// <summary>
+        /// Recognize Configurations.
+        /// </summary>
+        public RecognizeOptions RecognizeOptions { get; }
+
+        /// <summary>
+        /// Should stop current Operations?.
+        /// </summary>
+        public bool? InterruptCallMediaOperation { get; set; }
+
+        /// <summary>
+        /// Operation Context.
```

Comment:
How about "Customer supplied operation context, often used to identify this distinct operation. This property is sent back on the callback events related to this operation"?

The above description greatly helps the developer understand the "what" and the "why" in this case. Otherwise, "operation context" leaves me wondering why I care?!?

Reply:
I'll update in an upcoming PR.